oomph/LinguaFranca.setup

```diff
@@ -353,19 +353,19 @@
         <requirement
             name="de.cau.cs.kieler.klighd.lsp"/>
         <requirement
-            name="com.fasterxml.jackson.core.jackson-core" />
+            name="com.fasterxml.jackson.core.jackson-core"/>
         <requirement
-            name="com.fasterxml.jackson.core.jackson-annotations" />
+            name="com.fasterxml.jackson.core.jackson-annotations"/>
         <requirement
-            name="com.fasterxml.jackson.core.jackson-databind" />
+            name="com.fasterxml.jackson.core.jackson-databind"/>
         <repositoryList
             name="DefaultRepositories">
           <repository
-              url="http://download.eclipse.org/releases/2021-09"/>
+              url="http://download.eclipse.org/releases/2022-03"/>
           <repository
-              url="http://download.eclipse.org/tools/orbit/downloads/drops/R20210825222808/repository/"/>
+              url="http://download.eclipse.org/tools/orbit/downloads/drops/R20220302172233/repository/"/>
```

Comment:
@a-sr: where does this magic URL come from? I'm guessing this has to do with the upgrade from Eclipse 21-09 to 22-03, but when we upgrade to 22-06, how do we find the correct repository URL to list here?

Reply:
The Eclipse Orbit a repository of auxiliary plugins, simultaneously released with eclipse.
I get the magic URL (with the correct timestamp for each release) from this page: https://download.eclipse.org/tools/orbit/downloads/

Reply:
That's very useful info. Thanks, @a-sr!